chef/cookbooks/hyperv/attributes/default.rb

```diff
@@ -2,11 +2,31 @@
 # Windows features required:
 #
 
-default[:features_list][:hyperv] = ["Microsoft-Hyper-V", "Microsoft-Hyper-V-Management-PowerShell"]
-default[:features_list][:management] = ["Remote-Desktop-Services"]
-default[:features_list][:iscsi_target] = ["File-Services", "CoreFileServer"]
+default[:features_list][:hyperv] = {
+  "Microsoft-Hyper-V" => {
+    "restart" => false
+  },
+  "Microsoft-Hyper-V-Management-PowerShell" => {
+    "restart" => true
```

Comment:
@ovisan I'm worried it could interrupt chef-client while it's doing something, like unpacking the python archive we ship. In that last case, chef-client might not try to unpack again if some files are already extracted.

I might be too careful, though. It's just that I have no idea how this interacts with the chef run. I guess I'd somehow prefer to have us stop the chef run as soon as we know that we request a restart.